### ISSUE TITLE
Combobox: accessibility violation about aria-activedescendant

### ISSUE THREAD

**Issue by brunano21** [ibm-js/deliteful#658]
When the popup is open, the Combobox's `input` gets the `aria-activedescendant` attribute. 
Running an RPT scan a violation gets raised, reporting:
 
    The d-combobox element with WAI-ARIA property aria-activedescendant
    set to d-combobox-item-1 does not have tabindex attribute set to 0 or -1.

However setting the `tabindex` attribute to the `input` element is not enough yet. In order to fix the violation both `d-combobox` and `d-combobox-input` node must have the `tabindex` attribute set.

**Comment by wkeese** [comment]
>  assign renderer.renderNode (instead of renderer.childNodes[0]) to list.navigatedDescendant.

Makes sense.

>  both d-combobox and d-combobox-input node must have the tabindex attribute set

We try to only have one tabindex per form control, so can you just set `aria-activedescendant` on the `<input>` rather than the `<d-combobox>`?   Generally all the aria roles should be on the focused node anyway.

**Comment by wkeese** [comment]
PS: I split the first bug off to #662.   Since the bug database is used to generate the report of what changed in a release, each bug should have a separate ticket.

**Comment by wkeese** [comment]
Closing this for now.  The error RPT is reporting may just be a problem with RPT.

**Comment by brunano21** [comment]
It's confirmed it was an invalid RPT violation.
